Add CADD courses @ SB on the website (#129)
volkamerlab/volkamerlab_org: content/education/cadd-sb.md

---

> title: CADD courses & seminars @ SB

Reviewer: ```suggestion
title: CADD courses & seminars @ UdS
```

date: 2023-04-25
publishdate: 2023-04-25
menu:
  main:
    parent: Education
    name: CADD courses @ SB
weight: 20
nav: false
---

> We offer courses and seminars focused mainly on data-driven drug design in the [Center for Bioinformatics at Saarland University](https://zbi-www.bioinf.uni-sb.de/en/). Our role is to teach and provide supervision to students in diverse individual projects.

Reviewer: ```suggestion
We offer courses and seminars focused mainly on data driven drug design embedded in the [Center for Bioinformatics at Saarland University](https://zbi-www.bioinf.uni-sb.de/en/). We also supervise students from bioinformatics, as well as computer and life sciences, in diverse individual research projects.
```

<!--more-->

> Our [CADD course](https://www.lsf.uni-saarland.de/qisserver/rds?state=verpublish&status=init&vmfile=no&publishid=140645&moduleCall=webInfo&publishConfFile=webInfo&publishSubDir=veranstaltung) is mainly a hands-on lecture, students learn various methods, including available software tools and libraries, that support data generation and analysis in different areas of computer-aided drug design (CADD). 

Reviewer: ```suggestion
Our [CADD course](https://www.lsf.uni-saarland.de/qisserver/rds?state=verpublish&status=init&vmfile=no&publishid=140645&moduleCall=webInfo&publishConfFile=webInfo&publishSubDir=veranstaltung) is an interactive lecture, in which students learn various methods from structural bio- and cheminformatics, including available software tools and libraries, that support data generation and analysis in different areas of computer-aided drug design (CADD). 
```

> The course sensitizes students to the straightforward and profitable use of bio-/cheminformatics methods. It is designed to enable students to build their own CADD pipeline, including protein target preparation, compound library assembly and filtering, protein-ligand docking and result evaluation. The course composed of lectures to explain the various theories, a brief introduction to Python programming using interactive Jupyter notebooks and exercises in which students develop their own scripts, leaning on [TeachOpenCADD]( https://github.com/volkamerlab/teachopencadd).

Reviewer: ```suggestion
The course sensitizes students to the straightforward and profitable use of bio-/cheminformatics methods. It is designed to enable students to build their own CADD pipeline, including protein target preparation, compound library assembly and filtering, protein-ligand docking and result evaluation. The course is composed of lectures to explain the various concepts, a brief introduction to Python programming using interactive Jupyter notebooks and exercises in which students develop their own scripts, leaning on [TeachOpenCADD]( https://github.com/volkamerlab/teachopencadd).
```

> Our [CADD seminar](https://www.lsf.uni-saarland.de/qisserver/rds?state=verpublish&status=init&vmfile=no&publishid=144048&moduleCall=webInfo&publishConfFile=webInfo&publishSubDir=veranstaltung) , is an introductory seminar on computer-aided drug design, including structure- and ligand-based methods. After giving a brief introduction, students select a topic from the presented drug development  pipeline and independently acquire knowledge on the selected topics in the context of structural bio- and cheminformatics. 

Reviewer: ```suggestion
Our [CADD seminar](https://www.lsf.uni-saarland.de/qisserver/rds?state=verpublish&status=init&vmfile=no&publishid=144048&moduleCall=webInfo&publishConfFile=webInfo&publishSubDir=veranstaltung), is an introductory seminar to computer-aided drug design, including structure- and ligand-based methods. After giving a brief introduction, students select a topic from the presented drug development pipeline and independently acquire knowledge on the selected topics in the context of structural bio- and cheminformatics. 
```

> Topics include homology modeling, target assessment, molecular docking, virtual screening, molecular fingerprints, chemical similarity, ADMET predictions and QSAR; especially with increased focus on ML and DL methods. Under guidance, material ( papers, data and algorithms ) are collected by each student and presented at the end of the seminar, including the implementation of an application of the individual topics.

Reviewer: ```suggestion
Topics include homology modeling, target assessment, molecular docking, virtual screening, molecular fingerprints, chemical similarity, ADMET predictions and QSAR; especially with increased focus on ML and DL methods. Under guidance, material (papers, data and algorithms ) are collected by each student and presented at the end of the seminar, including the implementation of an application of the individual topics.
```

- CADD Seminar (144048): Data Driven Drug Design - Methods and Applications - <a href=" https://www.lsf.uni-saarland.de/qisserver/rds?state=verpublish&status=init&vmfile=no&publishid=144048&moduleCall=webInfo&publishConfFile=webInfo&publishSubDir=veranstaltung" target="_blank" class="external">SS2023</a>
- CADD Course (140645): Open-Source Software and Libraries for Data Driven Drug Design  - <a href=" https://www.lsf.uni-saarland.de/qisserver/rds?state=verpublish&status=init&vmfile=no&publishid=140645&moduleCall=webInfo&publishConfFile=webInfo&publishSubDir=veranstaltung" target="_blank" class="external">WS2022</a>

## Resources

> - 2023 - <a class="icon fa-lock" target="_blank" href="https://github.com/volkamerlab/CADDSeminar_2023" title="Access restricted to enroled students"> CADD Seminar</a> + <a class="icon fa-lock" target="_blank" href="https://github.com/volkamerlab/CADDSeminar_2023" title="Access restricted to enroled students"> Seminar</a>

Reviewer: ```suggestion
- SoSe 2023 - <a class="icon fa-lock" target="_blank" href="https://github.com/volkamerlab/CADDSeminar_2023" title="Access restricted to enrolled students"> CADD Seminar</a>
```

> - 2022 - <a class="icon fa-lock" target="_blank" href="https://github.com/volkamerlab/CADDCourse2023" title="Access restricted to enroled students"> CADD course</a> + <a class="icon fa-lock" target="_blank" href="https://github.com/volkamerlab/CADDCourse2023" title="Access restricted to enroled students"> Course</a>

Reviewer: ```suggestion
- WiSe2022/23 - <a class="icon fa-lock" target="_blank" href="https://github.com/volkamerlab/CADDCourse2023" title="Access restricted to enrolled students"> CADD course</a>
```
